cclib/method/nuclear.py

```diff
@@ -104,14 +104,15 @@ def repulsion_energy(self, atomcoords_index: int = -1) -> float:
         """Return the nuclear repulsion energy."""
         nre = 0.0
         for i in range(self.data.natom):
-            ri = self.data.atomcoords[atomcoords_index][i]
+            atomcoords = self.data.atomcoords[atomcoords_index]
```

Comment:
I know what's going on now, sorry for the confusion.  If you place this outside the loop (since it doesn't depend on the loop variable `i`) and wrap it in `convertor(..., "Angstrom", "bohr")`, ...

Reply:
I mean have
```python
nre = 0.0
atomcoords = convertor(self.data.atomcoords[atomcoords_index], "Angstrom", "bohr")
for i in range(self.data.natom):
...
```
and then getting rid of `/ convertor(d, "Angstrom", "bohr")`.

Reply:
Thank you!